test/e2e/autoscaler_test.go

```diff
@@ -0,0 +1,55 @@
+//go:build e2e
+// +build e2e
+
+/*
+Copyright 2023 The Kubernetes Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package e2e
+
+import (
+	. "github.com/onsi/ginkgo/v2"
+	"k8s.io/utils/pointer"
+)
+
+var _ = Describe("When using the autoscaler with Cluster API [Autoscaler] [PR-Blocking]", func() {
```

Comment:
Made this [PR-Blocking] only for testing to bypass some flakiness. Will drop it later.

Reply:
The PR is close to merge. Dropping it.